service-base/src/main/java/org/eclipse/hono/service/auth/X509CertificateChainValidator.java

```diff
@@ -39,4 +38,18 @@
      * @throws IllegalArgumentException if the chain is empty.
      */
     Future<Void> validate(List<X509Certificate> chain, TrustAnchor trustAnchor);
+
+    /**
+     * Validates a certificate path based on a list of trust anchors.
+     * 
+     * @param chain The certificate chain to validate. The end certificate
+     *              must be at position 0.
+     * @param trustAnchors The list of trust anchors to use for validating the chain.
+     * @return A completed future if the certificate path is valid for any of the trust anchors.
```

Comment:
Can we please use the same text as that of the other method variant? There is no reason why this method should behave differently, is there?